public/css/style.css

```diff
@@ -0,0 +1,85 @@
+/*
+*
+* ==========================================
+* ==========================================
+*
+*/
+
```

Comment:
It is better to delete line from 1 to 6 at all :)

Reply:
ahh yess true, agreed with Oksana